src/index.html

```diff
@@ -20,8 +21,348 @@
       href="styles/index.scss"
```

Comment:
The CSS file linked in the head section has the 'scss' extension, which is incorrect. The browser will not be able to interpret SCSS syntax. It should be a CSS file. You should compile your SCSS to CSS and then link the CSS file in your HTML.